php-checks/pom.xml

```diff
@@ -18,6 +18,25 @@
       <artifactId>php-frontend</artifactId>
     </dependency>
     <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>php-frontend</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+      <type>test-jar</type>
```

Comment:
@vilchik-elena The "usual "dependency of php-checks on php-frontend gives to php-checks test classes (e.g., to GenericExceptionCheckTest) visibility on the main classes of php-frontend. It does not give visibility on the test classes of php-frontend. As we need visibility on PHPCheckTest which is a test class of php-frontend, I had to add this test-jar dependency